### PR TITLE
[ADD] module checks the structured com of a supplier invoice

### PR DESCRIPTION
**Pull request by houssine78**
allows to check structured communication of a supplier invoice when the company is configured to generated bba com

### CODE REVIEW

File: account_invoice_check_bba_com/views/account_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        <field name="name">account.invoice.supplier.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_supplier_form"/>
+        <field name="arch" type="xml">
+        	<field name="reference" position="before">
+                <field name="bypass_check_bbacom"/>
+            </field>
+        </field>
+    </record>
+</odoo>
```
Comment (vvrossem) [inline account_invoice_check_bba_com/views/account_invoice.xml]: missing EOL

**Review by vvrossem — CHANGES_REQUESTED** [review]
LGTM.
Does it replace PR https://github.com/coopiteasy/cie-imported-modules/pull/3/ ?
Could you also add `readme` directory and `.rst` ?

**Comment by houssine78** [comment]
> 
> 
> LGTM.
> Does it replace PR [coopiteasy/cie-imported-modules#3](https://github.com/coopiteasy/cie-imported-modules/pull/3) ?
> Could you also add `readme` directory and `.rst` ?

yes it replaces indead those modules. I'll do it

**Comment by houssine78** [comment]
@vvrossem readme directory and .rst files was already there. Did you was asking for something particular ?

**Review by vvrossem — APPROVED** [review]

**Pull request merged**